oat-sa/extension-tao-itemqti
# Hotfix/CON-305/CON-540/ fix adding interaction by plus button - plugin blockAdder

Related to: https://oat-sa.atlassian.net/browse/AUT-195

Backport: https://github.com/oat-sa/extension-tao-itemqti/pull/1604 https://github.com/oat-sa/extension-tao-itemqti/pull/1624 https://github.com/oat-sa/extension-tao-itemqti/pull/1625

Related to: https://oat-sa.atlassian.net/browse/CON-305
https://oat-sa.atlassian.net/browse/CON-285
https://oat-sa.atlassian.net/browse/CON-540

How to test:

- Enable plugin in config/taoQtiItem/qtiCreator.conf.php option 'multi-column' => true,
- Go to Item Authoring
- Add any interaction to the canvas
- Click "+" icon to add text block
- Add any Inline interaction to just added text block
- UI shouldn't break
- try to double click on plus button
- in opened widget check some Common interaction
- plugin should work correctly, no errors in console
